sdks/uniswapx-sdk/src/utils/OrderQuoter.ts

```diff
@@ -155,6 +158,15 @@ async function checkTerminalStates(
           order.order.info.nonce
         );
         return cancelled ? OrderValidation.NonceUsed : OrderValidation.Expired;
+      }
+      // if the order has block overrides AND order validation is OK, it is invalid if current block number is < block override
+      else if (order.order.blockOverrides && validation === OrderValidation.OK) {
```

Comment:
nit: can `checkDefined(order.order.blockOverrides)` above this block to avoid having to do `!`

Reply:
oh looks like i dont even need the !